File: C/eulerOne.c
#include <stdio.h>

int main(void) {
    int i, sumThree = 0, sumFive = 0, sumBoth = 0, sum;
    for (i = 1; i <= 999; i++) {
        if (i % 3 == 0) {
            sumThree += i;
        }
        if (i % 5 == 0) {
            sumFive += i;
        }
        if (i % 15 == 0) {
            sumBoth += i;
        }
        if (i == 999) {
            sum = sumThree + sumFive - sumBoth;
            printf("The sum of all the numbers less that 1000 that are multiples of 3 or 5 is %d\n", sum);
        }
    }
}

File: Python/1 - 10/eulerOne.py
sumThree, sumFive, sumBoth = 0, 0, 0
for i in range(1000):
    if i % 3 == 0:
        sumThree = sumThree + i
    if i % 5 == 0:
        sumFive = sumFive + i
    if i % 15 == 0:
        sumBoth = sumBoth + i
print("The sum of all the numbers less that 1000 that are multiples of 3 or 5 is", sumThree + sumFive - sumBoth)